post-scan-actions/aws-python-publish-malware-to-sns/template.yml

```diff
@@ -0,0 +1,90 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cloudone-filestorage-plugin-publish-malware-to-sns
+    Description: >-
+      According to the scan result from the scanner, publish the malware information
+      to the target SNS topic.
+    Author: Trend Micro Cloud One File Storage Security
+
+Parameters:
+  TargetSnsTopicName:
+    Type: String
+    Description: The name for the SNS topic to which malware information publish.
+  PublishMalwareLambdaName:
+    Type: String
+    Description: The name for the lambda function publishes the malware information to SNS topic.
+  PublishMalwarePolicyName:
+    Type: String
+    Description: The name for the policy includes sns and account authority.
+  PublishMalwareRoleName:
+    Type: String
+    Description: The name for the role to execute the lambda function.
```

Comment:
For internal use resources, we usually don't let the customer choose the names. There might be problems, like name already existed, that would block customer from deploying the template. Any purpose for these parameters?

Reply:
I would change them into a `Prefix` parameter and set the resources name as `{prefix}-{region}-malware-plugin`. Do you think this is a good way to name the resources?

Reply:
I will remove these and using the default name (stack name).